.changeset/honest-apricots-fail.md

```diff
@@ -0,0 +1,5 @@
+---
+'@shopify/cli-hydrogen': patch
+---
+
+Added `h2 upgrade` command
```

Comment:
I assume there will be some documentation that we can link to in the release notes?